post-policy.go

```diff
@@ -206,6 +206,28 @@ func (p *PostPolicy) SetUserMetadata(key string, value string) error {
 	return nil
 }
 
+// SetUserData - Set user data as a key/value couple.
+// Can be retrieved through a HEAD request or an event.
+func (p *PostPolicy) SetUserData(key string, value string) error {
+	if strings.TrimSpace(key) == "" || key == "" {
```

Comment:
If `key==""` is `true` then `strings.TrimSpace(key) == ""` will also be `true`

Reply:
removed `strings.TrimSpace(key) == ""`